index.html

```diff
@@ -13,158 +13,175 @@
+      <nav>
+        <img id="menu" src="imgs/Union.svg" />
+        <ul class="navbar">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="c1">
+        <h1>
+          <span class="clr-blue"
+            >Hey there. <br class="hide" />I'm Blesslin.<br
+          /></span>
+          <span class="clr-green">
+            I'm a Software <br class="hide" />Developer</span
+          >
+        </h1>
+        <p class="content-1-p">
+          I can help you build a product , feature or website Look through some
+          of my work and<br class="hide-mob" />
+          experience! If you like what you see and have a project you need
+          coded,<br class="hide-mob" />don’t hesitate to contact me.
+        </p>
+        <div class="flex" id="social-media">
+          <img src="imgs/socialmedia.png" alt="social media" />
+        </div>
+        <img
+          id="d"
+          src="https://img.icons8.com/external-dreamstale-lineal-dreamstale/32/null/external-down-arrow-arrows-dreamstale-lineal-dreamstale-2.png"
+        />
       </div>
-      <img
-        id="d"
-        src="https://img.icons8.com/external-dreamstale-lineal-dreamstale/32/null/external-down-arrow-arrows-dreamstale-lineal-dreamstale-2.png"
-      />
     </section>
 
     <section class="works">
       <section id="content-2">
-        <h1 class="clr-blue">My Recent <br />Works</h1>
+        <h1 class="clr-blue">My Recent <br class="hide" />Works</h1>
         <img src="imgs/Indicator.png" alt="indicator" />
       </section>
       <!-- cards -->
-      <section class="s-card">
-        <div id="card">
-          <div class="card1">
-            <h3>Multi-Post Stories <br />Gain + Glory</h3>
-            <ul class="li">
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">HTML</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">CSS</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">Javascript</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">Ruby</li>
-              &nbsp;&nbsp;&nbsp;
-            </ul>
-            <button class="btn">See project</button>
+      <div class="works-flex">
+        <section class="s-card">
+          <div id="card">
+            <div class="card1">
+              <h3>Multi-Post Stories <br />Gain + Glory</h3>
+              <ul class="li">
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">HTML</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">CSS</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">Javascript</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">Ruby</li>
+                &nbsp;&nbsp;&nbsp;
+              </ul>
+              <button class="btn">See project</button>
```

Comment:
- [ ] Please make sure you follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=dWWSbacCPQ0hVj82-0)🎨 for styling the button here. This ensures you follow a given design to perfection which is a quality trait for a developer like yourself.
      ![firefox_kz6PEcoelc](https://user-images.githubusercontent.com/104968236/228802829-d7b93b1c-a0d0-4458-8a22-86209b80a4f2.png)
    - Make sure you have the proper `padding` as well as the `:hover` and `:active `state